Tests/Grid/Mapping/Metadata/ManagerTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace APY\DataGridBundle\Grid\Tests\Mapping\Metadata;
+
+use APY\DataGridBundle\Grid\Mapping\Column;
+use APY\DataGridBundle\Grid\Mapping\Driver\Annotation;
+use APY\DataGridBundle\Grid\Mapping\Metadata\Manager;
+use APY\DataGridBundle\Grid\Mapping\Metadata\Metadata;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Constraints\DateTime;
+
+class ManagerTest extends TestCase
+{
+    public function testAddDriverWithStrings()
+    {
+        $manager = new Manager();
+        $manager->addDriver('foo', 'bar');
```

Comment:
`addDriver` needs a priority so `bar` does not seem a representative value. Even `foo` does not seem a proper value, you should use a mock of `DriverInterface`